test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsTests.cs

```diff
@@ -2242,8 +2242,8 @@ public void GetConfigFilePaths_ReadsFilesCorrectly()
         [Fact]
         public void GetConfigFilePaths_SettingsWithoutFiles_ReturnEmptyList()
         {
-            var settings = new Settings(settingsHead: null);
-
+            var settings = new Settings(new List<SettingsFile>());
+            // Why does this not throw?
```

Comment:
I also want to know.

Reply:
😆 Fixing it now...it was a remnant of an old implementation :D